addons/stock/models/stock_orderpoint.py

```diff
@@ -306,6 +306,10 @@ def _get_orderpoint_action(self):
         """
         action = self.env["ir.actions.actions"]._for_xml_id("stock.action_orderpoint_replenish")
         action['context'] = self.env.context
+        self.cron_orderpoint_cleanup()
+        return action
+
+    def cron_orderpoint_cleanup(self):
```

Comment:
You don't have to create a new cron for this one. You can decorate your new function with the autovacuum decorator
https://www.odoo.com/documentation/14.0/reference/orm.html#module-odoo.api (check odoo.api.autovacuum). It will be part of the daily odoo vacuum cron